doc/source/ray-core/doc_code/direct_transport_gloo.py

```diff
@@ -127,10 +127,10 @@ def sum(self, tensor: torch.Tensor):
 # passed back to the same actor without copying.
 sum1 = sender.sum.remote(tensor)
 sum2 = receiver.sum.remote(tensor)
-assert torch.allclose(*ray.get([sum1, sum2]))
+assert torch.allclose(*ray.get([sum1, sum2], _tensor_transport="object_store"))
 # __gloo_intra_actor_end__
 
 # __gloo_get_start__
-print(ray.get(tensor))
+print(ray.get(tensor, _tensor_transport="object_store"))
```

Comment:
Can you add an example here showing that plain `ray.get` will fail with an exception? And update direct-transport.rst to explain that you need to pass in _tensor_transport here.

Reply:
done